### ISSUE TITLE
[metis] Build error

### ISSUE THREAD

**Issue by KEITA-spec** [microsoft/vcpkg#25970]
Package: metis[core]:x64-windows -> 5.1.0#8
**Host Environment**
-  Host: x64-windows
- Compiler: MSVC 19.29.30141.0
-    vcpkg-tool version: 2022-07-21-a0e87e227afb536c62188c11ad029954f28fdb22
    vcpkg-scripts version: f93ba152d 2022-07-22 (2 days ago)

**To Reproduce**
`vcpkg install opencv4 ceres ceres[suitesparse] lapack suitesparse`

I followed the OpenSfM Building procedure.

I got an error when I entered this code.
 ` vcpkg install opencv4 ceres ceres[suitesparse] lapack suitesparse --triplet x64-windows`

[https://opensfm.org/docs/building.html](url)
1.Download: 

> git clone --recursive https://github.com/mapillary/OpenSfM

2.Install dependencies:

>  pip install -r requirements

3.Installing dependencies on Windows

> cd OpenSfM
> git clone https://github.com/microsoft/vcpkg
> cd vcpkg
> bootstrap-vcpkg.bat
> vcpkg install opencv4 ceres ceres[suitesparse] lapack suitesparse --triplet x64-windows

**Failure logs**

error: building metis:x64-windows failed with: BUILD_FAILED
Then check for known issues at:
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[metis]+Build+error
    vcpkg-tool version: 2022-07-21-a0e87e227afb536c62188c11ad029954f28fdb22
    vcpkg-scripts version: f93ba152d 2022-07-22 (2 days ago)

You can also use the prefilled template from C:\Users\k99i0\source\OpenSfM\vcpkg\installed\vcpkg\issue_body.md.

```
-- Note: metis only supports static library linkage. Building static library.
-- Downloading http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz -> metis-5.1.0.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' = off
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Downloading http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz
[DEBUG] Download failed -- retrying after 1000 ms.
[DEBUG] Download failed -- retrying after 2000 ms.
[DEBUG] Download failed -- retrying after 4000 ms.
Error: Failed to download from mirror set:
http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz: WinHttpSendRequest() failed: 12002
http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz: WinHttpSendRequest() failed: 12002
http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz: WinHttpSendRequest() failed: 12002
http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz: WinHttpSendRequest() failed: 12002

[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(766): 
[DEBUG] Time in subprocesses: 0 us
[DEBUG] Time in parsing JSON: 3 us
[DEBUG] Time in JSON reader: 0 us
[DEBUG] Time in filesystem: 3061 us
[DEBUG] Time in loading ports: 0 us
[DEBUG] Exiting after 1.526 min (91518128 us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:24 (message):
      
      Failed to download file with error: 1
      If you use a proxy, please check your proxy setting. Possible causes are:
      
      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.
      
      2. You are using Fiddler. Currently a bug (https://github.com/microsoft/vcpkg/issues/17752)
         will set HTTPS_PROXY to `https://fiddler_address:port` which lead to problem 1 above.
         Workaround is open Windows 10 Settings App, and search for Proxy Configuration page,
         Change `http=address:port;https=address:port` to `address`, and fill the port number.
      
      3. Your proxy's remote server is out of service.
      
      In future vcpkg releases, if you are using Windows, you no longer need to set
      HTTP(S)_PROXY environment variables. Vcpkg will simply apply Windows IE Proxy
      Settings set by your proxy software. See (https://github.com/microsoft/vcpkg-tool/pull/49)
      and (https://github.com/microsoft/vcpkg-tool/pull/77)
      
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:249 (z_vcpkg_download_distfile_show_proxy_and_fail)
  ports/metis/portfile.cmake:6 (vcpkg_download_distfile)
  scripts/ports.cmake:147 (include)



```
**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

**Comment by KEITA-spec** [comment]
I don't know how to solve it.

**Comment by LilyWangLL** [comment]
Duplicate to #25764. `metis` will be fixed in PR #25953.
